VAMobile/src/screens/HealthScreen/SecureMessaging/SecureMessaging.test.tsx

```diff
@@ -69,8 +95,13 @@ jest.mock('../../../api/authorizedServices/getAuthorizedServices', () => {
 })
 
 context('SecureMessaging', () => {
-  const initializeTestInstance = () => {
-    render(<SecureMessaging {...mockNavProps()} />)
+  const initializeTestInstance = (errorsState = initialErrorsState) => {
+    render(<SecureMessaging {...mockNavProps()} />, {
+      preloadedState: {
+        ...InitialState,
+        errors: errorsState,
```

Comment:
had to update the error mock to include states, no idea why this was working before either